### ISSUE TITLE
Support for typescript ?

### ISSUE THREAD

**Issue by Epistol** [northosts/wuxt#100]
Could be nice to have full support of typescript

**Comment by northosts** [comment]
agreed, not a priority right now, though. you are welcome to join development with some PRs ;)
